bootstrap/kubeadm/internal/controllers/kubeadmconfig_controller_test.go

```diff
@@ -1099,20 +1133,28 @@ func TestBootstrapTokenTTLExtension(t *testing.T) {
 	} {
 		result, err := k.Reconcile(ctx, req)
 		g.Expect(err).ToNot(HaveOccurred())
-		g.Expect(result.RequeueAfter).NotTo(BeNumerically(">=", k.TokenTTL))
+		g.Expect(result.RequeueAfter).To(Equal(k.TokenTTL / 3))
```

Comment:
I consider `TTL / 3` much more than just an implementation detail by now and therefore test for the exact value. It's important that the `RequeueAfter` interval leads to multiple reconciliation attempts before the TTL expires, so testing for `0 < result.RequeueAfter < TokenTTL` isn't sufficient.